OG/OG/AstVisiting/Visitor interfaces/Visitor interfaces/ICoodinateXyVisitor.cs

```diff
@@ -0,0 +1,10 @@
+﻿using OG.ASTBuilding.Terminals;
+
+namespace OG.AstVisiting
+{
+    public interface ICoodinateXyVisitor
+    {
+        public void Visit(CoordinateXyValueNode node);
+        public void Visit(OG.ASTBuilding.TreeNodes.BodyNodesAndVisitors.CoordinateXyValueNode node);
```

Comment:
hvorfor begge to?

Reply:
Fordi begge findes og bliver brugt i AST'et. Så det er for ikke at ændre i AST.